roxie/ccd/ccdfile.cpp

```diff
@@ -1059,9 +1065,12 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
             const char *kind = fdesc.queryKind();
             if (kind && streq(kind, "key"))
                 isKey = true;
+            StringBuffer planeName;
+            fdesc.getClusterLabel(pdesc->copyClusterNum(0), planeName);
```

Comment:
I'm not sure if this is the right call to get the plane name that this file lives on?

Reply:
Should be good to do this way.